test/soft-redirects.js

```diff
@@ -67,7 +67,9 @@ describe('Soft Redirects', function() {
 
 });
 
-describe('Soft Redirects - with `statusCode` option', async() => {
+describe('Soft Redirects - with `statusCode` option', async function() {
+
+  this.timeout(t.timeout);
```

Comment:
Default timeout was too short every time today, versus the preferred timeout that we set in all our other test cases (in the same way I'm doing it here).

I think our startups are getting slower, but that's not a problem for this morning.